inc/class-s3-uploads-wp-cli-command.php

```diff
@@ -403,6 +407,29 @@ private function recurse_copy($src,$dst) {
 		}
 		closedir($dir);
 	}
+
+	/**
+	 * Verify that the required constants for the S3 connections are set.
+	 *
+	 * @return bool true if all constants are set, else false.
+	 */
+	private function verify_s3_access_constants() {
```

Comment:
@fklein-lu in the case of `S3_UPLOADS_USE_INSTANCE_PROFILE` being true, the constants for key / secret are not actually required. Could you create an issue and patch that, thanks!